controller/handler/handler.go

```diff
@@ -3,43 +3,49 @@ package handler
 import (
 	"net/http"
 
-	"github.com/TwiN/gatus/v4/config/ui"
-	"github.com/TwiN/gatus/v4/security"
+    "github.com/TwiN/gatus/v4/config"
 	"github.com/TwiN/health"
 	"github.com/gorilla/mux"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
-func CreateRouter(staticFolder string, securityConfig *security.Config, uiConfig *ui.Config, enabledMetrics bool) *mux.Router {
+func CreateRouter(staticFolder string, config *config.Config) *mux.Router {
```

Comment:
Could you rename the parameter `config` to `cfg` so it doesn't collide with the `config` package?

Reply:
Resolved by #307